### PR TITLE
Created the Contact form of the mobile version

### PR DESCRIPTION
**Pull request by Bellagirl-maker**
Portfolio: add a contact form and finish the mobile version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -171,6 +171,16 @@ <h4>2015</h4>
             </ul>
     </section>
 </div>
+<section class="contact-me">
+    <p class="contact-intro1">Contact me</p>
+    <p class="contact-intro2">If you have an application you <br>are interested in developing, a<br> feature that you need built or a <br>project that needs coding. I’d <br>love to help with it </p>
+    <form action="https://formspree.io/f/{form_id}" method="post">
```
Comment (leonardodiasb) [inline index.html]: - [ ] Kindly create a [Formspree](https://formspree.io/html/) account and add your id to this form action so you can receive user messages through this form. Your form is very similar to the template, but without your id, it is not useful.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Bellagirl-maker, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Correct use of GitHub flow. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Bellagirl-maker) [reply index.html]: @leonardodiasb I have created the account but I don't see my id.

Comment (Bellagirl-maker) [reply index.html]: @leonardodiasb I have created an account on Formspree and added my id to this form as requested

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Bellagirl-maker 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - No linter errors. ✔️
 - Correct use of GitHub flow. ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**